packages/rumdl/package.yaml

```diff
@@ -0,0 +1,18 @@
+---
+name: rumdl
+description: Fast Markdown linter and formatter written in Rust.
+homepage: https://github.com/rvben/rumdl
+licenses:
+  - MIT
+languages:
+  - Markdown
+categories:
+  - Linter
+  - Formatter
+  - LSP
+
+source:
+  id: pkg:cargo/rumdl@0.0.164
```

Comment:
while using `cargo` is not wrong per se, it is preferable to use github releases instead, since that avoids requiring the rust toolchain as a dependency on the user side. (and as opposed to node / python, most users do not have cargo installed in your system)